pkg/reconciler/v1alpha1/revision/revision.go

```diff
@@ -324,6 +324,8 @@ func (c *Reconciler) reconcileDigest(ctx context.Context, rev *v1alpha1.Revision
 }
 
 func (c *Reconciler) reconcile(ctx context.Context, rev *v1alpha1.Revision) error {
+	// TODO: Revision should inherit autoscaling annotations from Namespace, like it does from Configuration.
```

Comment:
The KPA is a namespaced resource, so IMO the Revision shouldn't inherit them, but the KPA should be sensitive to them.  Does that make sense?

Reply:
I'll pull out the namespace inheritance and open a wider discussion about how we should address this use case.

Reply:
Asked the question here: https://github.com/knative/serving/issues/2428